aten/src/ATen/gen.py

```diff
@@ -360,6 +403,28 @@ def filter_by_extension(files, *extensions):
     return filtered_files
 
 
+def generate_per_op_registration(per_op_registrations):
+    if not options.per_op_registration:
+        return
+
+    # Ensure all whitelisted operators have a corresponding registration file.
+    # Generate an empty placeholder file for nonexistent operators, which might
+    # be registered manually instead of via codegen.
```

Comment:
Hmm, why is the empty placeholder file needed?

Reply:
It can simplify the BUCK side logic. We can unconditionally create BUCK targets for each operator (with srcs = [ pt_op_register_XXX.cpp ]), and unconditionally add them to the `deps` of feature targets. I'm not sure if there is easy way to make BUCK tolerant targets with nonexistent src files.

You may want to ask why cannot we ignore manually registered ops in the first place (before calling the script)?

A model might be calling regular autogenerated aten ops and/or manually registered ops (e.g. quantized ops), and these ops might be calling ops in the other category. For example:

- codegen registered ops calling manually registered ops:
aten::quantized_lstm -> quantized::linear_dynamic

- manually registered ops calling codegen registered ops:
quantized::relu6 -> aten::empty

Even though we cannot turn on/off the registration of quantized ops via codegen, we need take them into consideration when calculating what codegen registered ops need to be kept for custom build.

We could create some rule / metadata (e.g. `manual_kernel_registration` which doesn't include quantized ops thus incomplete) to handle this, but I feel an empty placeholder is probably the simplest solution.

Reply:
OK, makes sense to me. A good comment to put in the code so we remember it in the future :)

Reply:
Added more comments.